tests/test_polyspace.py

```diff
@@ -54,14 +57,17 @@ def test_bug_finder_tsv_file(self):
         with open(TEST_IN_DIR / 'polyspace.tsv', newline="") as file:
             with patch('sys.stdout', new=StringIO()) as fake_out:
                 self.warnings.check_logfile(file)
-                count = self.warnings.return_count()
+                count = self.warnings.return_check_limits()
         stdout_log = fake_out.getvalue()
-
-        for checker in self.dut.checkers:
-            self.assertIn(
-                f"{checker.count} warnings found for '{checker.column_name}': '{checker.check_value}'",
-                stdout_log
-            )
+        self.assertEqual(
+            '\n'.join([
+                "polyspace: family 'defect'               information: impact: high     number of warnings (42) is higher than the maximum limit (0).",
+                "polyspace: family 'defect'               information: impact: medium   number of warnings (9) is higher than the maximum limit (0).",
+                "polyspace: family 'defect'               information: impact: low      number of warnings (4) is higher than the maximum limit (0).",
```

Comment:
There's a lot of redundant padding.

```suggestion
                "polyspace: family 'defect' | information: impact: high   | number of warnings (42) is higher than the maximum limit (0).",
                "polyspace: family 'defect' | information: impact: medium | number of warnings (9) is higher than the maximum limit (0).",
                "polyspace: family 'defect' | information: impact: low    | number of warnings (4) is higher than the maximum limit (0).",
```

Reply:
Resolved in cf49d1a6817418d74f047d30d9946df680be87b4